### DISCUSSION TITLE
Why can't we mint to any address we want?

### DISCUSSION THREAD

**Discussion by Ifra001** [Cyfrin/foundry-full-course-cu#2738] [Q&A]
I am in the markle airdrop part and in the ```deployment script```, this is the code I am a little confused about
```solidity
       token.mint(token.owner(), s_amountToTransfer);
        token.transfer(address(airdrop), s_amountToTransfer);
```
Why are we first minting to the ```owner``` of the token and then transferring to the ```airdrop``` contract. why can't the airdrop contract directly mint like this:
```solidity
token.mint(address(airdrop), s_amountToTransfer);
```

**Comment by EngrPips** [comment]
Hello, Thats a good question to ask. Honestly I didn't see any reason why we can't do it as you have suggested. Try it and let us know if it works out fine.

**Reply by Ifra001** [reply]
sure, I'll try it and if there is any bug, I will report it here!

**Reply by EngrPips** [reply]
I am looking forward to it.
